server.js

```diff
@@ -184,6 +204,7 @@ function initialise () {
   const app = unconfiguredApp
   if (NODE_ENV !== 'test') {
     app.use(metrics.initialise())
+    app.use(paths.csp.path, cspMiddlewareStack) // CSP violation monitoring
```

Comment:
As this is setting up a particular endpoint that executes a middleware stack, rather than us binding middleware to a bunch of endpoints, I think this would be easier to understand and find if we moved this to `routes.js` and used `app.post` to set up the endpoint.

Reply:
good suggestion, have amended